core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -133,6 +133,16 @@ private[spark] class ExecutorAllocationManager(
 
   private val defaultProfileId = resourceProfileManager.defaultResourceProfile.id
 
+  private val streamingDRAFeatureEnabled = conf.get(DYN_ALLOCATION_STREAMING_ENABLED)
```

Comment:
it would be nice to update the description of this class to include the streaming details and how the algorithm works.

Reply:
Sounds good. Will add more info at class description. But will add more documentation with example in https://spark.apache.org/docs/latest/job-scheduling.html section.